### DISCUSSION TITLE
Laser not wolking later update version 4.3.0

### DISCUSSION THREAD

**Discussion by Llfernandez** [arkypita/LaserGRBL#1415] [Q&A]
I have a cnc3018 laser with a 1000mw laser after updating the lasergrbl to version 4.3.0, it starts to burn and 2 seconds later I only see the blue light but it does not burn. that could be happening. please help.

**Comment by arkypita** [comment]
Your laser is died, and the update of lasergrbl has nothing to do with it.

**Comment by Llfernandez** [comment]
laser works well with universal gcodec.
I attach my configuration files, are they correct?

Which Laser model do you recommend for the 3018 cnc, it has 450nm out power
1000mW

**Reply by arkypita** [reply]
Good to know your laser is not dead.

Attachment does not work when reply to github email but your configuration is the same if you use universal gcode or lasergrbl because it is stored inside the controller.
